tools/state-parts/src/cli.rs

```diff
@@ -20,12 +20,12 @@ pub struct StatePartsCommand {
 
     #[clap(long)]
     /// genesis hash to use in the Handshake we send. This must be provided if --chain-id
-    /// is not "mainnet" or "testnet"
+    /// is not MAINNET or TESTNET
     genesis_hash: Option<String>,
 
     #[clap(long)]
     /// head height to use in the Handshake we send. This must be provided if --chain-id
-    /// is not "mainnet" or "testnet"
+    /// is not MAINNET or TESTNET
     head_height: Option<u64>,
```

Comment:
I would revert this one, especially if it's case sensitive.

Reply:
Done, thank you!